postman/collection-dir/nmi/Flow Testcases/Happy Cases/Scenario10-Create a mandate and recurring payment/Payments - Create/event.test.js

```diff
@@ -0,0 +1,89 @@
+pm.environment.set("random_number", _.random(100, 100000));
```

Comment:
Shouldn't we be doing this as part of the pre-request script?

Reply:
Not necessarily since it takes the value before sending the request.